### ISSUE TITLE
vue 3

### ISSUE THREAD

**Issue by PremKolar** [BKWLD/vue-in-viewport-directive#24]
hi, is there any way to get this running with Vue 3?
I've tried to no avail :(

**Comment by 3birdsmedia** [comment]
Yes, I love this plugin but Quasar has recently moved to vue3 and all of our projects are heading that direction.

**Comment by weotch** [comment]
It will definitely come in the future.  We use this all the time still, but we're heavy Nuxt users and that hasn't added Vue 3 support yet, so we haven't felt any _internal_ pressure to update yet.

**Comment by 3birdsmedia** [comment]
That makes sense! Hopefully it comes sooner than later 😄 
Would you consider doing the work if I convince my company to pay for the work it would take?
